dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilderTest.java

```diff
@@ -124,4 +125,22 @@ public void testSetDetectedOsSystemProperties_netty4Dependency() throws Reposito
     List<Artifact> artifacts = DependencyGraphBuilder.getDirectDependencies(nettyArtifact);
     Truth.assertThat(artifacts).isNotEmpty();
   }
+
+  @Test
+  public void testSetDetectedOsSystemProperties_grpcProtobufExclusion() throws RepositoryException {
+    // Grpc-protobuf depends on grpc-protobuf-lite with protobuf-lite exclusion.
+    // https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1056
+    Artifact nettyArtifact = new DefaultArtifact("io.grpc:grpc-protobuf:1.25.0");
+
+    DependencyGraph dependencyGraph =
+        DependencyGraphBuilder.getStaticLinkageCheckDependencyGraph(
+            ImmutableList.of(nettyArtifact));
+
+    Truth.assertThat(dependencyGraph.list())
```

Comment:
This is hard to follow. Can it be spit out into several statements?

Reply:
Extracted Correspondence into a variable.